src/App.tsx

```diff
@@ -1,109 +1,29 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Post } from './types/Post';
+
+function getUserById(userId: number) {
```

Comment:
Add type to function return

```suggestion
function getUserById(userId: number): User | null {
```